.github/workflows/release-buildx.yml

```diff
@@ -9,12 +9,12 @@ on:
     - cron: '0 1 * * 0' # every sunday at 1am
   workflow_dispatch:
     inputs:
-      repo:
-        description: 'Override repo'
-        required: false
-        type: string
       ref:
-        description: 'Override ref'
+        description: 'Ref (e.g. v0.10.0)'
+        required: true
```

Comment:
Now version/ref is required when creating a release. Previously it takes the default one being set in Makefile/Bake definition.